### DISCUSSION TITLE
[THEME] Basic light and dark themes

### DISCUSSION THREAD

**Discussion by brianwisti** [TriliumNext/Trilium#4302] [Show and tell]
Wanted to do something with the Notion color palette. It's still very much in progress, but I like it enough to share at this point.

https://github.com/brianwisti/trilium-basic-themes

A few specific adjustments for my ADHD brain: it sets colors for italic and bold text, and uses [Atkinson Hyperlegible](https://brailleinstitute.org/freefont) if you have it on your system.

Dark theme:

![basic-theme-dark](https://github.com/zadam/trilium/assets/161212/0419893f-a916-4c73-bc52-d8439b3c86a6)

Light theme:

![basic-theme-light](https://github.com/zadam/trilium/assets/161212/467c849d-170b-44b0-a5a3-40a6eae44a19)

**Comment by jacobxperez** [comment]
@brianwisti Hello, I have noticed that you are using Atkinson Hyperlegible font, I recommend upgrading to [Atkinson Hyperlegible Pro](https://github.com/jacobxperez/atkinson-hyperlegible-pro) because the original typeface is not being updated or maintain. I have added 199 extra glyph's supporting 91 languages including Vietnamese.

**Comment by rodrigoalexandrecosta** [comment]
Very nice theme!
